editor/src/components/editor/import-utils.ts

```diff
@@ -212,3 +223,54 @@ export function getImportsFor(
 
   return emptyImports()
 }
+
+export function removeUnusedImports(
+  element: JSXElement,
```

Comment:
Minor point here, but I'd suggest renaming this to `removedElement`, and possibly even renaming the function to reflect that this is removing unused imports specifically for that removed element